ssh_filter_btrbk.sh

```diff
@@ -23,6 +23,21 @@ file_match_sane='/[0-9a-zA-Z_@+./-]*' # matches file path (equal to ${file_match
 file_match="/[^']*" # btrbk >= 0.32.0 quotes file arguments: match all but single quote
 file_arg_match="('${file_match}'|${file_match_sane})" # support btrbk < 0.32.0
 
+is_pathname_absolute()
```

Comment:
See previous comment, I don't think we should contstrain too much here. This just makes the code more complex, and implies that `//` has some special meaning for btrbk, which it does not.
A simple check for leading slash might make some sense for crazy users trying to specify relative paths for whatever reason. Even then, the error is just shifted to here, without that check it will fail saying that the path does not match.

Reply:
It may not have in btrbk, but it may in principle have in the OS... so a OS could choose to handle `//` not like `/` but e.g. like and object identifier.<br>
Then all the constraints we set up to make things secure, wouldn't work anymore. Thus I think it's better to simply forbid `//` at all. In practise no one should anyway ever set that.

Reply:
Sadly btrbk does exactly this in some cases (which are not uncommon), see fix: 142fa6c982c3996a46a24e8de8d5c530ef03022b

This means we should keep supporting leading `//`, at least before next major release.

Unless it really imposes a security risk, but: I don't know of any linux (!) OS where `//` is treated in a special way, but maybe you know better. OS other that linux will not install ssh_filter_btrbk.sh, as btrfs is only available on linux as far as I know.